--- hwclock/rtc.c       2011-09-26 11:50:22.000000000 +0200
+++ hwclock/rtc.c       2015-08-02 23:26:28.990576490 +0200
@@ -267,7 +267,7 @@
 #else
                rc = ioctl(rtc_fd, RTC_UIE_ON, 0);
 #endif
-               if (rc == -1 && (errno == ENOTTY || errno == EINVAL)) {
+               if (1 || rc == -1 && (errno == ENOTTY || errno == EINVAL)) {
                        /*
                         * This rtc device doesn't have interrupt functions.
                         * This is typical on an Alpha, where the Hardware

